src/services/ldap/strategies/univention.js

```diff
@@ -22,85 +22,95 @@ class UniventionLDAPStrategy extends AbstractLDAPStrategy {
+	 * @memberof UniventionLDAPStrategy
+	 */
 	getUsers(school) {
 		const options = {
 			filter: 'univentionObjectType=users/user',
 			scope: 'sub',
-			attributes: [
-				'givenName', 'sn', 'mailPrimaryAdress', 'mail', 'dn', 'entryUUID', 'uid', 'objectClass', 'memberOf',
-			],
+			attributes: ['givenName', 'sn', 'mailPrimaryAdress', 'mail', 'dn', 'entryUUID', 'uid', 'objectClass', 'memberOf'],
 		};
 		const searchString = `cn=users,ou=${school.ldapSchoolIdentifier},${this.config.rootPath}`;
-		return this.app.service('ldap').searchCollection(this.config, searchString, options)
-			.then((data) => data.map((obj) => {
-				const roles = [];
-				if (obj.objectClass.includes('ucsschoolTeacher')) {
-					roles.push('teacher');
-				}
-				if (obj.objectClass.includes('ucsschoolStudent')) {
-					roles.push('student');
-				}
-				if (obj.objectClass.includes('ucsschoolStaff')) {
-					// toDo
-				}
+		return this.app
+			.service('ldap')
+			.searchCollection(this.config, searchString, options)
+			.then((data) =>
+				data.map((obj) => {
+					const roles = [];
+					if (obj.objectClass.includes('ucsschoolTeacher')) {
+						roles.push('teacher');
+					}
+					if (obj.objectClass.includes('ucsschoolStudent')) {
+						roles.push('student');
+					}
+					if (obj.objectClass.includes('ucsschoolStaff')) {
+						// toDo
+					}
 
-				return {
-					email: obj.mailPrimaryAddress || obj.mail,
-					firstName: obj.givenName,
-					lastName: obj.sn,
-					roles,
-					ldapDn: obj.dn,
-					ldapUUID: obj.entryUUID,
-					ldapUID: obj.uid,
-				};
-			}));
+					return {
+						email: obj.mailPrimaryAddress || obj.mail,
+						firstName: obj.givenName,
+						lastName: obj.sn,
+						roles,
+						ldapDn: obj.dn,
+						ldapUUID: obj.entryUUID,
+						ldapUID: obj.uid,
+					};
+				})
+			);
 	}
 
 	/**
-     * @public
-     * @see AbstractLDAPStrategy#getClasses
-     * @returns {Array} Array of Objects containing className, ldapDn, uniqueMembers
-     * @memberof UniventionLDAPStrategy
-     */
+	 * @public
+	 * @see AbstractLDAPStrategy#getClasses
+	 * @returns {Array} Array of Objects containing className, ldapDn, uniqueMembers
+	 * @memberof UniventionLDAPStrategy
+	 */
 	getClasses(school) {
 		const options = {
 			filter: `ucsschoolRole=school_class:school:${school.ldapSchoolIdentifier}`,
 			scope: 'sub',
 			attributes: [],
 		};
-		const searchString = 'cn=klassen,cn=schueler,cn=groups,'
-			+ `ou=${school.ldapSchoolIdentifier},${this.config.rootPath}`;
-		return this.app.service('ldap').searchCollection(this.config, searchString, options)
-			.then((data) => data.map((obj) => {
-				const splittedName = obj.cn.split('-');
-				return {
-					className: splittedName[splittedName.length - 1],
-					ldapDn: obj.dn,
-					uniqueMembers: obj.uniqueMember,
-				};
-			}));
+		const searchString =
+			'cn=klassen,cn=schueler,cn=groups,' + `ou=${school.ldapSchoolIdentifier},${this.config.rootPath}`;
```

Comment:
Codacy found an issue: [Unexpected string concatenation of literals.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6074601)